AdminUi/src/AdminUi/ClientApp/src/app/utils/odata-response.ts

```diff
@@ -0,0 +1,3 @@
+export interface ODataResponse<Type> {
+    value: Type;
```

Comment:
Should be same as in the "normal" API.
```suggestion
    result: Type;
```